package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "leaderboard-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack serve --mode development",
+    "build": "webpack --mode production"
```

Comment:
- [ ] Nice job so far. I recommend you add the "start" script here so that users can easily start the app using the `npm run start` command on the terminal as instructed in your readme. :pray: 

![Screenshot from 2021-11-09 22-34-28](https://user-images.githubusercontent.com/64894927/140992389-d08a6e01-1acf-465b-86d6-0366f9a6307e.png)